internal/microsoft_edge.py

```diff
@@ -68,8 +68,20 @@ def prepare(self, job, task):
         self.bodies_path = os.path.join(task['dir'], 'bodies')
         if not os.path.isdir(self.bodies_path):
             os.makedirs(self.bodies_path)
-        if not task['cached']:
-            self.clear_cache()
+        try:
+            import _winreg
+            keyval = r"SOFTWARE\Classes\Local Settings\Software\Microsoft\Windows\CurrentVersion\AppContainer\Storage\microsoft.microsoftedge_8wekyb3d8bbwe\MicrosoftEdge\Privacy"
+            if not os.path.exists("keyval"):
```

Comment:
AFAIK, os.path.exists checks the filesystem (relative to the cwd), not the registry and this will always be false.